tests/view/view/jumpoveruielement.js

```diff
@@ -34,12 +35,14 @@ describe( 'View', () => {
 	}
 
 	beforeEach( () => {
+		stylesProcessor = new StylesProcessor();
```

Comment:
This should be inlined. ps.: I've jumped over a few files as the page slows down on this big PR :D